### ISSUE TITLE
Model Training

### ISSUE THREAD

**Issue by Mahnoor200377** [NaVi-start/Sign-IDD#4]
Hi

I hope you are well.

While reviewing your code and the /Configs/Sign-IDD.yaml  file, I noticed that the model was trained for 20,000 epochs until the learning rate dropped below 0.0002. However, when I trained the model using the same code and configuration, the training stopped after approximately 37 epochs due to minimum learning rate.

Could you kindly clarify why there is a difference? Is there a possibility that the code or configuration files you used are different from the ones shared? Thank you for your time and support.
